src/applications/personalization/profile/components/personal-health-care-contacts/PersonalHealthCareContacts.jsx

```diff
@@ -0,0 +1,131 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import {
+  selectProfileContactsToggle,
+  selectProfileContacts,
+  selectEmergencyContact,
+  selectNextOfKin,
+} from '@@profile/selectors';
+import { fetchProfileContacts } from '@@profile/actions';
+import { focusElement } from '~/platform/utilities/ui';
+
+import Contact from './Contact';
+import Loading from './Loading';
+import LoadFail from '../alerts/LoadFail';
+
+const HelpDeskContact = () => (
+  <>
+    <va-telephone data-testid="va-800-number" contact="8006982411" /> (
+    <va-telephone data-testid="va-711-number" contact="711" tty="true" />)
+  </>
+);
```

Comment:
Also non-blocking, but I think usually it is preferred to use the constants that are provided from the component library to reference phone numbers so they can be updated if need be easily from one place.

Here is an example https://github.com/department-of-veterans-affairs/vets-website/blob/main/script/github-actions/daily-product-scan/tests/mocks/applications/app-1/components/GetFormHelp.jsx#L4